vochain/transaction/election_tx.go

```diff
@@ -301,3 +333,22 @@ func (t *TransactionHandler) txElectionCostFromProcess(process *models.Process)
 		MaxVoteOverwrite:        process.GetVoteOptions().MaxVoteOverwrites,
 	})
 }
+
+// txCostIncreaseCensusSize calculates the cost increase of a process based on the new census size.
+func (t *TransactionHandler) txCostIncreaseCensusSize(process *models.Process, newSize uint64) uint64 {
```

Comment:
I think maxCensusSize naming was a bad decision, it has led to a lot of confusion.

In this case "censusSize" looks friendlier. I think we must s/maxCensusSize/censusSize but thid would break compatibility. At some point we need to do a refactor on protobuf models naming, but not now.

So this is why I named it "censusSize", I know it is not consistent, but its the correct naming IMHO.